cinema/urls.py

```diff
@@ -1 +1,23 @@
-# write urls here
+from django.urls import include, path
+from rest_framework import routers
+
+from cinema.views import (
+    ActorViewSet,
+    CinemaHallViewSet,
+    GenreViewSet,
+    MovieSessionViewSet,
+    MovieViewSet,
+)
+
+
+router = routers.DefaultRouter()
+router.register("cinema_halls", CinemaHallViewSet, basename="cinema_hall")
+router.register("actors", ActorViewSet)
+router.register("genres", GenreViewSet)
+router.register("movies", MovieViewSet)
+router.register("movie_sessions", MovieSessionViewSet)
+
+
+urlpatterns = [path("", include(router.urls))]
```

Comment:
No need to include if basepath is not provided. Instead use:
urlpatterns = router.urls